### DISCUSSION TITLE
Mail Configuration

### DISCUSSION THREAD

**Discussion by sparthe123** [Status-Page/Status-Page#451] [Q&A] [answered]
Hi Team, 

I added email configuration in configuration.py, but subscription verification emails are not working.. and also there is no log in mail server from the application. Is there any other place I need to configure mail setup 

Where can I check the application logs ?

**Comment by herrtxbias** [comment]
You only need to configure the parameters in the `configuration.py`. Did you restart the services after you have changed the parameters? Could you share your configuration here, so I can have a look and see if there are any issues?

**Comment by BornTKill** [comment]
Dear,

Same issue on my side.
However, python manage.py sendtestemail —admins is working.

**Comment by sparthe123** [comment]
I did try restart after changing values in configuration.py, but no luck.. 

Attaching mail configuration code below, please let me know if you need more details. 

# Email settings
EMAIL = {
    'SERVER': 'smtp.myserver.co',
    'PORT': 'xxxx',
    'USERNAME': 'xxxxx',
    'PASSWORD': 'xxxxx',
    'USE_SSL': False,
    'USE_TLS': False,
    'TIMEOUT': 10,  # seconds
    'FROM_EMAIL': 'status@domian.com',
}

**Comment by herrtxbias** [comment]
Please check for failed background tasks in the Django Admin Panel. You should see an entry in the "default" queue with the callable "utilities.utils.send_mail". There you may find more information.

**Comment by sparthe123** [comment]
I couldnt find that option from Django Admin panel, When I click Background task option it shows only RQ Queues statistics. 

attaching screenshot 
https://share.anysnap.app/fJHyk22w6GQT

**Comment by herrtxbias** [comment]
Thats correct, if you click on the number in the column "failed" for the row "default" you would see if there is any failed job. Since there is no failed job, you may have to try sending a mail. If there is still no failed job, you may have to check the logs of your smtp server for connection attempts.

**Comment by BornTKill** [comment]
Dear @HerrTxbias ,

I do not find any utilities.utils.send_mail job in the default queue.

`/opt/status-page/statuspage/manage.py rqworker high default low -v 3` does not help.

**Comment by sparthe123** [comment]
Hi @HerrTxbias , yeah I checked the smtp logs, there is no request from the application.

**Comment by BornTKill** [comment]
In subscribers/models.py I modified line 76/77:

```
        #if not self.email_verified_at:
            #return None
```

And email is received.

because in subscribers/views.py we have line 29

`        if action == 'resend_verification_mail' and not subscriber.email_verified_at:
`

**Comment by BornTKill** [comment]
And I now have a finished job utilities.utils.send_mail

**Comment by herrtxbias** [comment, marked as answer]
I've opened an issue for this bug:
- #468
